### PR TITLE
fix: support end col for eslint

### PR DESCRIPTION
**Pull request by sQVe**
This PR aims to add end_col support for the `eslint` and `eslint_d` linters. 

See https://github.com/mfussenegger/nvim-lint/issues/394 for background.

### CODE REVIEW

**Comment by sQVe** [comment]
Do note that I haven't had the time to verify if the altered test passes locally, since it's getting late. I would preferably verify it via CI, if possible.

**Comment by sQVe** [comment]
Ouff, I thought I could rerun the tests after you approved them. I'll make sure to make the tests pass locally before making any further pushes. 👍🏼

**Comment by sQVe** [comment]
Tests pass locally now 👍🏼

File: lua/lint/linters/eslint.lua [hunk 1]
```diff
@@ -1,27 +1,47 @@
...
+  vim.diagnostic.severity.ERROR,
+  vim.diagnostic.severity.WARN,
 }
 
 return require('lint.util').inject_cmd_exe({
   cmd = function()
-    local local_eslint = vim.fn.fnamemodify('./node_modules/.bin/eslint', ':p')
-    local stat = vim.loop.fs_stat(local_eslint)
+    local local_eslintd = vim.fn.fnamemodify('./node_modules/.bin/eslint', ':p')
+    local stat = vim.loop.fs_stat(local_eslintd)
```
Comment (mfussenegger) [inline lua/lint/linters/eslint.lua:10]:
```suggestion
    local local_eslint = vim.fn.fnamemodify('./node_modules/.bin/eslint', ':p')
    local stat = vim.loop.fs_stat(local_eslint)
```

File: lua/lint/linters/eslint.lua [hunk 2]
```diff
@@ -1,27 +1,47 @@
...
     if stat then
-      return local_eslint
+      return local_eslintd
```
Comment (mfussenegger) [inline lua/lint/linters/eslint.lua:12]:
```suggestion
      return local_eslint
```

**Pull request merged**